lambdas/main.go

```diff
@@ -278,20 +279,56 @@ func statusHandler(ctx context.Context, req StatusRequest) (interface{}, error)
 	var clusterStatus interface{}
 	var err error
 	if req.Type == "status" {
-		clusterStatus, err = status.GetClusterStatus(ctx, stateTable, stateTableHashKey, stateKey)
+		clusterStatus, err = getClusterStatus(ctx, stateTable, stateTableHashKey, stateKey)
 	} else if req.Type == "progress" {
 		clusterStatus, err = status.GetReports(ctx, stateTable, stateTableHashKey, stateKey, clusterName, hostGroup)
 	} else {
 		clusterStatus = "Invalid status type"
 	}
 
 	if err != nil {
+		log.Error().Err(err).Send()
 		return "Failed retrieving status: %s", err
 	}
 
 	return clusterStatus, nil
 }
 
+func getClusterStatus(ctx context.Context, stateTable, stateTableHashKey, stateKey string) (protocol.ClusterStatus, error) {
+	clusterStatus, err := status.GetClusterStatus(ctx, stateTable, stateTableHashKey, stateKey)
+	if err != nil {
+		return protocol.ClusterStatus{}, fmt.Errorf("getClusterStatus > %w", err)
+	}
+
+	clusterName := os.Getenv("CLUSTER_NAME")
+	if clusterName == "" {
+		return protocol.ClusterStatus{}, fmt.Errorf("CLUSTER_NAME is not set")
+	}
+	ips, err := common.GetBackendsPrivateIps(clusterName, "backend")
+	if err != nil {
+		return protocol.ClusterStatus{}, fmt.Errorf("getClusterStatus > GetBackendsPrivateIps: %w", err)
+	}
+	log.Info().Msgf("GetClusterStatus > Backend private IPs: %v", ips)
+
+	managementLambdaName := os.Getenv("MANAGEMENT_LAMBDA")
+	if managementLambdaName == "" {
+		return protocol.ClusterStatus{}, fmt.Errorf("MANAGEMENT_LAMBDA is not set")
+	}
+	managementRequest := management.ManagementRequest{
```

Comment:
no need to provide the backend ips in the request we can just fetch them in the function that gets the weka status instead of here

Reply:
I'll test this, but doesn't this.  I had assumed that it suffered from the same endpoint restrictions as the DynamoDB status lookup.

Reply:
actually you are probably right